packages/e2e/integration/attendance_spec.ts

```diff
@@ -12,127 +12,149 @@ import { slots } from "../__testData__/slots.json";
 const gus = customers["gus"];
 
 describe("AddAttendedCustomersDialog", () => {
-  beforeEach(() => {
-    // Initialize app, create default user,
-    // create default organization, sign in as admin
-    cy.initAdminApp()
-      // Save organziation to cypress context (making it available to the test functions)
-      .then((organization) => cy.wrap(organization).as("organization"))
-
-      // Load test data into firestore
-      .then((organization) =>
-        cy.updateCustomers(organization, customers as Record<string, Customer>)
-      )
-      .then((organization) =>
-        cy.updateSlots(organization, slots as Record<string, SlotInterface>)
-      )
-
-      .then(() => cy.signIn())
-      .then(() =>
-        // Our test slot's date is "2022-01-01" so we want to set the clock to that date to open it immediately
-        cy.setClock(DateTime.fromISO("2022-01-01").toMillis())
-      )
-      .then(() => cy.visit(PrivateRoutes.Root));
-  });
+  describe("Add athletes dialog", () => {
+    beforeEach(() => {
+      // Initialize app, create default user,
```

Comment:
Why is this split into two different `describe` blocks with the exact same `beforeEach`?
Did it affect the CI matrix tests that way?